src/ext/php7/engine_hooks.c

```diff
@@ -170,7 +170,8 @@ static bool dd_should_trace_helper(zend_execute_data *call, zend_function *fbc,
                                    scope ? "::" : "", Z_STRVAL(fname));
             }
 
-            zval retval, *integration = &dispatch->deferred_load_integration_name;
+            zval retval = {.u1.type_info = IS_UNDEF};
```

Comment:
Should we make a similar change in [handlers_curl.c](https://github.com/DataDog/dd-trace-php/blob/9a38904447dcd4ccb913194c30776726a84180de/src/ext/php7/handlers_curl.c#L269-L272)?

Reply:
That case won't segsigv without this PR because it doesn't use the fn_proxy. With this PR if it did use the fn_proxy it would still not sigsegv because I changed ddtrace_call_function to initilaize the retval like zend_call_function would.

So... yes, we probably should, but it's not an issue presently.